x/stakeibc/ibc_middleware.go

```diff
@@ -107,6 +107,16 @@ func (im IBCMiddleware) OnChanOpenAck(
 	if err != nil {
 		return err
 	}
+	communityPoolDepositOwner := types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_COMMUNITY_POOL_DEPOSIT)
```

Comment:
yeah let's revisit this later. I'm not convinced using an empty string to indicate it would be the best route (but I'm also not convinced using a flag is the best route either).

I think we should just start with having each host be the same and then we can try and filter later